pyogrio/core.py

```diff
@@ -217,6 +217,14 @@ def read_info(
     driver or if the data source is nonspatial. You can force it to be calculated using
     the ``force_total_bounds`` parameter.
 
+    ``fid_column`` is the name of the FID field in the data source, if the FID is
+    physically stored (e.g. in GPKG). If the FID is just a sequence, ``fid_column``
+    will be "" (e.g. ESRI Shapefile).
+
+    ``geometry_name`` is the name of the field where the main geometry is stored in the
```

Comment:
Potentially confusing aspect here is that `read_dataframe` actually does not honor that name, but always uses "geometry"

Reply:
Presumably it should?  Which wouldn't be a breaking change, because GeoPandas would also return it as the `.geometry` attribute of a GeoDataFrame, regardless of the name we use for the geometry field when constructing the GeoDataFrame?

Reply:
Users still do `df["geometry"]` as well, in addition to `df.geometry`, so changing the geometry name we return in `read_dataframe` would be quite a big change, I think

Reply:
> Users still do `df["geometry"]` as well, in addition to `df.geometry`, so changing the geometry name we return in `read_dataframe` would be quite a big change, I think

I can confirm I use `df["geometry"]` regularly as well...